### PR TITLE
ILAMB task revision

### LINKED ISSUE #133: Add task for `e3sm_to_cmip`
Add task for `e3sm_to_cmip`

### LINKED ISSUE #134: Add task for ILAMB
Add task for ILAMB

### LINKED ISSUE #229: Support ILAMB on other machines
Support ILAMB on other machines. Follow-up from #197, #230.

### PR DESCRIPTION
**Pull request by forsyth2**
Add ILAMB task. Revision of #197. Resolves #133. Resolves #134. A future pull request will handle #229.

### CODE REVIEW

**Comment by forsyth2** [comment]
It took some debugging to create the single initial commit for this pull request. I've listed what I believe to be the necessary steps below:
```
$ git fetch upstream e3sm2cmip_ilamb
$ git checkout -b e3sm2cmip_ilamb upstream/e3sm2cmip_ilamb
$ git rebase --rebase-merges -i upstream/main
# There are 8 commits:
# pick
# change to fixup
# change to fixup
# change to fixup
# merge -C
# pick
# change to fixup
# change to fixup
# Now, there are 3 commits.
# See https://stackoverflow.com/questions/32840872/convert-merge-into-rebase-without-having-to-perform-the-merge-again
$ git reset --soft HEAD~2
$ git log
# Now, only the first commit shows up
$ git commit --no-verify # Effectively turns the merge commit (and the commit after) into a single regular commit
$ git rebase -i upstream/main
# There are 3 commits:
# pick
# change to fixup
# change to fixup
# Now, there is 1 commit.
$ git commit --amend
# change commit title to "ILAMB task initial commit"
$ git push forsyth2 e3sm2cmip_ilamb
# Create this pull request
# Confirmed that this pull request adds 1,580 lines and removes 2 lines, just as #197 does.
```

**Comment by forsyth2** [comment]
@chengzhuzhang This pull request contains two commits. The first is all the work you did on #197. The second is my revisions: https://github.com/E3SM-Project/zppy/pull/230/commits/32108949ab7be8fc65ff0c009cf46c61fea92e09 -- that's what needs a review. Once we merge this, I'll update the expected files for the integration tests.

Notable changes: 
- Added parameters (and defaults) for `ts_atm_subsection` and `ts_land_subsection`. This enables multiple `ts` subtasks to be run (e.g., with different values for `ts_fmt`).
- Changed location of setting `export` to `ALL`. 
- Made sure `cmip_metadata` and `ts_fmt` were defined in the appropriate places.
- Added code to move visuals to web server.
- Changed `tmp` directory name to avoid name collisions.

**Comment by forsyth2** [comment]
Also note I'll handle #229 in a follow-up pull request.

File: tests/integration/test_complete_run.cfg [hunk 1]
```diff
@@ -33,6 +33,12 @@ years = "1850:1854:2",
   input_files = "eam.h0"
   input_subdir = "archive/atm/hist"
 
+  [[ atm_monthly_180x360_aave_ilamb ]]
```
Comment (chengzhuzhang) [inline tests/integration/test_complete_run.cfg:36]: could this be [[ atm_monthly_180x360_aave ]]? I think the logic is if ts_fmt=cmip, both nco convention time series and cmip time series are generated.
And this sub session can be both a dependency of both e3sm_diags and ilamb.

**Review by chengzhuzhang — COMMENTED** [review]
Hey Ryan, the PR looks good, I only has one query left as a single line comment. Thanks!

File: tests/integration/test_complete_run.cfg (hunk 1 above)
Comment (forsyth2) [reply tests/integration/test_complete_run.cfg]: Thanks, it must have been some issue in debugging that made me think it required a separate subtask. It looks like it works fine using the same subtask.

**Review by chengzhuzhang — APPROVED** [review]

**Comment by forsyth2** [comment]
Expected files for integration tests (https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.forsyth2/zppy_test_complete_run_www/) have been updated.

**Pull request merged**